metatensor-torch/tests/neighbor-checks/slanted-loose.json

```diff
@@ -0,0 +1,207 @@
+{
+    "description": "Half NL for a basic FCC lattice",
+    "system": {
+        "positions": [
+            [1.42, 0.0, 0.0],
+            [2.84, 0.0, 0.0],
+            [3.55, -1.22975607, 0.0],
+            [4.97, -1.22975607, 0.0]
+        ],
+        "cell": [
+            [4.26, -2.45951215, 0.0],
+            [2.13, 1.22975607, 0.0],
+            [0.0, 0.0, 50.0]
+        ]
+    },
+    "options": {
+        "cutoff": 4.1,
+        "full_list": false,
+        "strict": false
```

Comment:
I'm not sure what this is intended to test? The set of pairs included in a non-strict NL will depend a lot on the actual implementation. I think these files are more about checking the strict case, and in the non-strict case the set of `expected-neighbors` is the minimal set of pairs that should be included.